package.json

```diff
@@ -81,7 +81,8 @@
     "typescript": "^3.8.3",
     "unist-util-visit": "^2.0.3",
     "unist-util-visit-parents": "^3.1.1",
-    "yargs": "^16.0.0"
+    "yargs": "^16.0.0",
+    "detect-indent": "^7.0.0"
```

Comment:
`detect-indent 7.x` requires at least Node.js 12.20, what can we do?

Reply:
does typescript have a way of conditionally importing and otherwise stubbing (such that pre-12.20 users just don't get this feature?)

Reply:
TypeScript directly, no. But it is always possible to do an if/else. But I think that using the second last version (6.1.0) is a better solution. The last version (7.0.0) only brings some minor and syntax changes and the use of ESM.